Makefile

```diff
@@ -1,17 +1,21 @@
-all: lkmpg.tex
-	rm -rf _minted-main
-	pdflatex -shell-escap lkmpg.tex
-	bibtex main >/dev/null || echo
+PROJ = lkmpg
+all: $(PROJ).pdf
+
+$(PROJ).pdf: lkmpg.tex
+	rm -rf _minted-$(PROJ)
+	pdflatex -shell-escap $<
+	bibtex $(PROJ) >/dev/null || echo
 	pdflatex -shell-escape $< 2>/dev/null >/dev/null
 
 html: lkmpg.tex html.cfg
-	make4ht --shell-escape --utf8 --format html5 --config html.cfg --output-dir html lkmpg.tex
+	sed $'s/\t/    /g' lkmpg.tex > lkmpg-for-ht.tex
+	make4ht --shell-escape --utf8 --format html5 --config html.cfg --output-dir html  lkmpg-for-ht.tex
 	ln -sf lkmpg.html html/index.html
-	rm -f  lkmpg.xref lkmpg.tmp lkmpg.html lkmpg.css lkmpg.4ct lkmpg.4tc lkmpg.dvi lkmpg.lg lkmpg.idv lkmpg*.svg lkmpg.log lkmpg.aux 
+	rm -f  lkmpg-for-ht.tex lkmpg.xref lkmpg.tmp lkmpg.html lkmpg.css lkmpg.4ct lkmpg.4tc lkmpg.dvi lkmpg.lg lkmpg.idv lkmpg*.svg lkmpg.log lkmpg.aux
```

Comment:
I meet problem when make html files:
```bash
sed s/\t/    /g' lkmpg.tex > lkmpg-for-ht.tex
/bin/sh: 1: Syntax error: Unterminated quoted string
make: *** [Makefile:11: html] Error 2
```
And add an space between `$` and `'` worked for me:
```diff
--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,7 @@ $(PROJ).pdf: lkmpg.tex
        pdflatex -shell-escape $< 2>/dev/null >/dev/null
 
 html: lkmpg.tex html.cfg
-       sed $'s/\t/    /g' lkmpg.tex > lkmpg-for-ht.tex
+       sed $ 's/\t/    /g' lkmpg.tex > lkmpg-for-ht.tex
             ^
```